meteor-common/src/main/java/dev/pixelib/meteor/base/errors/InvocationTimedOutException.java

```diff
@@ -38,12 +38,4 @@ public String getMethodName() {
     public String getNamespace() {
         return namespace;
     }
-
-    /**
-     * @return Amount of time in seconds that the invocation was allowed to take.
-     */
-    public int getTimeoutSeconds() {
```

Comment:
This isn't used internally, which is correct, but that doesn't mean that it's useless.
Meteor is a library which is made to be consumed by other projects, and `InvocationTimedOutException` is an exception we may throw for when invocations time out.

`getTimeoutSeconds()` provides useful context for consuming projects, letting users implement custom logging or retry behaviour, because the lapse of `timeoutSeconds` is the core reason the exception is ever thrown.